File: Zadania z PDF'ów/Szyfr Cezara/c++/Podstawowy.cpp
#include <iostream>
#include <string>

using namespace std;

int main() {
    string jawny, szyfrogram = "";
    int klucz, i;
    int kod;
    cout << "Podaj tekst do zaszyfrowania: ";
    cin >> jawny;
    cout << "Podaj klucz: ";
    cin >> klucz;
    for (i = 0; i < jawny.size(); i++)
    {
        kod = jawny[i] + klucz;
        if (kod > 'Z') kod = kod - 26;
        szyfrogram += char(kod);
    }
    cout << "Szyfrogram: " << szyfrogram << endl;
    return 0;
}

File: Zadania z PDF'ów/Szyfr Cezara/Python/Podstawowy.py
import string

jawny = ""
szyfrogram = ""
klucz = 0
kod = 0

jawny = str(input("Podaj tekst do zaszyfrowania: "))
klucz = int(input("Podaj klucz: "))

for i in range(0, len(jawny)):
    kod = ord(jawny[i]) + klucz
    if kod > ord('Z'):
        kod -= 26
    szyfrogram += chr(kod)

print(szyfrogram)